src/pokemon.c

```diff
@@ -2312,6 +2312,9 @@ static u8 BoxPokemon_IsShiny(BoxPokemon *boxMon)
 
 static inline BOOL Pokemon_InlineIsPersonalityShiny(u32 monOTID, u32 monPersonality) {
     return (((monOTID & 0xFFFF0000) >> 16) ^ (monOTID & 0xFFFF) ^ ((monPersonality & 0xFFFF0000) >> 16) ^ (monPersonality & 0xFFFF)) < 8;
+        return 1;
+
```

Comment:
Where did this change come from?

Reply:
No idea where that came from. Do a git blame?